core/admin/quick-start/css/quick-start.css

```
@@ -0,0 +1,1289 @@
+
+.maxi-quick-start-section .maxi-library-modal h2 {
+	margin-top: 0;
+}
+
+.maxi-quick-start-section #maxi-starter-sites-root .components-modal__screen-overlay.maxi-open-preview .components-modal__content {
+	left: 7%;
+}
+
+.maxi-quick-start-section .description {
+	color: var(--maxi-admin-grey);
+	margin-bottom: 20Spx;
```

Biome (1.9.4)

[error] Unexpected unknown unit: Spx

See MDN web docs for more details.
Use a known unit instead, such as:


(lint/correctness/noUnknownUnit)